README.md

```diff
@@ -536,6 +540,43 @@ We propose to give extensions the ability to select which partition key to use w
 It is worth noting that by allowing extension background contexts to load cookies across different partitions allows extensions to use partitioned cookies to store cross-site identifiers.
 This problem is discussed further in [Security and Privacy Considerations](#security-and-privacy-considerations).
 
+### First-Party CHIPS
```

Comment:
This all looks right to me, though I'm not the most familiar with this stuff.

I think we should also mention that the protection against CSRF afforded by this scheme is less than `SameSite=Lax` in that `evil.com` can still access the cookie if embedded by `lp.com`:

`lp.com` embeds `evil.com` which embeds `lp.com` -> the innermost frame has access to the cookie

This means that `lp.com` should be more careful about embedding untrusted content than if it used `SameSite=Lax`.

Reply:
Done.